src/specs/ShadowRegistry/index.ts

```diff
@@ -45,6 +45,7 @@ HybridShadowRegistry.add = (handle, styles) => {
         .filter(style => !style?.initial?.updater)
         .filter(style => style && Object.keys(style).length > 0)
         .flat()
```

Comment:
just wondering would it be better to do `.flat(Number.POSITIVE_INFINITY)`. wondering can styles be nested in multiple layers?

Reply:
We can do that. Issue was about nested undefined in the array. C++ side didn't like it.